conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosGovernanceOperations.scala

```diff
@@ -193,15 +197,15 @@ object TezosGovernanceOperations extends LazyLogging {
    * extract the numbers returning a collector object.
    */
   private def fillAggregates(
-      proposalsBlocks: Map[Block, ProtocolId],
+      proposalsBlocks: Map[Block, Option[ProtocolId]],
       listingsPerLevel: Map[Int, List[Voting.BakerRolls]],
       ballots: Map[Block, List[Voting.Ballot]],
       ballotCountsPerLevel: Map[Block, BallotsPerLevel],
       ballotCountsPerCycle: Map[Int, BallotsPerCycle],
       proposalProtocolCounts: Map[Block, Map[ProtocolId, Int]] //comes from individual operations on the block
```

Comment:
`proposalProtocolCounts` is misleading. It implies we are counting the proposal operation counts by protocol ID. A better name might be something like `proposalCountsByBlock`.